project/ISSUE-TRIAGE.md

```diff
@@ -0,0 +1,70 @@
+Triaging of Issues
+------------------
+
+Triaging issues is important. Triage solves several issues for resolving an issue in a timely manner. First, it helps to condense and convey the authors original purpose. It can be sometimes hard to explain in an issue what is actually happening for an action that an end user may perform.  
+The second is a proper triage of an issue gives a contributor the information they need before the commit to resolving it. Finally, fighting duplicate issues helps to lower the issue count and prevent duplicate discussions.
+
+Remember there are more ways than just code that you can contribute to an open source project and the community would really thank you if you spend spend time triaging issues.
+
+### Important first steps
+
+Before jumping into an issue very far make sure that the author of the issue has provided enough information so that you are able to make an educated recommendation on how this is to be triaged. Standard information that *must* be included in most issues are things such as:
+
+-	the output of `docker version`
+-	the output of `docker info`
+-	the output of `uname -a`
+-	a reproducible case if this is a bug, Dockerfiles FTW
+-	host distribution and version ( ubuntu 14.04, RHEL, fedora 21 )
+
+Additional information can be requested if needed. If an issue is totally unable to be triaged based on the author's provided information, kindly state that they will have to provide the above information and clarification on the problem. If the author does not respond with more information within the timespan of a week, kindly close the issue letting them know that they can reopen when they have time available to provide the additional information.
```

Comment:
> kindly state that they will have to provide the above information

An `more info required` label could be useful here; that way, the issue-list could be filtered on that label; any issue without feedback can be closed (automatically?) after two days